src/breeze/components/breeze_component.cr

```diff
@@ -1,4 +1,28 @@
 abstract class Breeze::BreezeComponent < Lucky::BaseComponent
   include Lucky::UrlHelpers
   alias HtmlProc = Proc(IO::Memory) | Proc(Nil)
+
+  def mount(component : Lucky::BaseComponent, *args, **named_args) : Nil
```

Comment:
All this could probably be copy/pasted into Lucky for https://github.com/luckyframework/lucky/issues/1427